packages/address-book/types/token.ts

```diff
@@ -10,7 +10,6 @@ interface Token {
   oracle?: 'tokens' | 'lps';
   bridge?: string;
   staked?: boolean;
-  //todo: add more risks
-  risks?: ReadonlyArray<'NO_TIMELOCK'>;
+  risks?: ReadonlyArray<string>;
```

Comment:
i would do `ReadonlyArray<'NO_TIMELOCK'  | 'LARGE_HOLDERS'>` and then keep adding types to avoid having random issues

Reply:
Changed as advised